### ISSUE TITLE
Playstation Dualshock 3 Connected, but can't pair

### ISSUE THREAD

**Issue by Handika997** [blueman-project/blueman#1922]
blueman: blueman-manager 2.3.4
BlueZ:
Distribution: Linux Mint 21 Vanessa
Desktop environment: Cinnamon

* [x ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done 
ℹ Please use some service like [Pastebin](http://pastebin.com/) or [GitHub Gist](https://gist.github.com/) to post your logs and keep the thread clean and readable. Make sure to describe the exact steps you took when producing it.

probably controller issue, since my controller is a counterfeit, but previously on Linux Mint 20.3 I managed to make it work using qtsixa driver, but since I upgraded to linux mint 21 I can't pair my controller, even though it's detected and connected, I've tried installing qtsixa like before but I got errors and unavailable files/directories that didn't happened on Mint 20.3

In summary:

- Upgraded to Linux Mint 21
- Controller detected and connected, but can't pair (detected as keyboard and requested to enter PIN number)
- tried to install qtsixa and sixpair to no avail (previously worked in Mint 20.3)

->

**Comment by infirit** [comment]
It may be because you don't have the bluez-plugin installed. I do have an original one i can try later but everything you need is in BlueZ and the kernel these days so no need for additional things like qtsixa

**Comment by infirit** [comment]
So the process

1. Connect the ps3 controller with a cable
2. Accept the authentication (always accept)
3. Done
4. Unplug

**Comment by Handika997** [comment]
> It may be because you don't have the bluez-plugin installed. I do have an original one i can try later but everything you need is in BlueZ and the kernel these days so no need for additional things like qtsixa

can you kindly tell me the steps please

**Comment by infirit** [comment]
The sixaxis plugin is location in the bluez-plugin directory, for me this is `/usr/lib/bluetooth/plugins/sixaxis.so`. The steps how to pair are already provied.
